src/components/FiltersBar/useFiltersBar.ts

```diff
@@ -20,8 +20,7 @@ export const useFiltersBar = () => {
   const canClearVideoLengthFilter =
     videoWhereInput?.duration_lte !== undefined || videoWhereInput?.duration_gte !== undefined
   const canClearOtherFilters = videoWhereInput?.hasMarketing_eq === false || videoWhereInput?.isExplicit_eq === false
-  const canClearCategoriesFilter =
-    videoWhereInput?.categoryId_in !== undefined && videoWhereInput?.categoryId_in?.length !== 0
+  const canClearCategoriesFilter = videoWhereInput?.category !== undefined
```

Comment:
without this, when you select and unselect category, clear button is not disabled
```suggestion
  const canClearCategoriesFilter = videoWhereInput?.category !== undefined || videoWhereInput?.category.id_in.length !== 0
```

Reply:
Adjusted your suggestion slightly, please check